content/en/blog/2024-12-28-introducing-the-pre-new-year-release-of-open-source-platform-cozystack-v0-21.md

```
@@ -0,0 +1,52 @@
+title: "Introducing the Pre-New Year Release of open source platform Cozystack v0.21:"
+slug: introducing-the-pre-new-year-release-of-open-source-platform-cozystack-v0-21-
+date: 2024-12-28
+author: "Timur Tukaev"
+description: "The dashboard now works directly with the Cozystack API instead of relying on FluxCD resources. This enhancement enables the platform to…"
+---
+
+### Introducing the Pre-New Year Release of open source platform Cozystack v0.21: New User Dashboard, Talos Linux, etc.
+
+The dashboard now works directly with the Cozystack API instead of relying on FluxCD resources. This enhancement enables the platform to provide a user-friendly graphical interface while integrating with Kubernetes’ standard RBAC model for managing deployment permissions.
+
+![](https://cdn-images-1.medium.com/max/800/1*O0OQMDGX0oHS2AXm0zDg4g.png)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)